torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3136,6 +3136,19 @@ struct to_ir {
         return emitBuiltinCall(
             tree->range(), *method.graph(), kind, named_values, {});
       }
+      case '%': {
+        auto first_arg = emitSugaredExpr(Expr(tree->tree(0)), 0)->asValue(tree->tree(0)->range(), method);
+        auto first_arg_type = first_arg->type()->kind();
+        auto first_arg_type_string = std::string(typeKindToString(first_arg_type));
+        if (type_string.compare("StringType") == 0) {
+          auto values = getValues(tree->trees(), /*maybe_unpack=*/false);
+          auto node = graph->create(aten::percentFormat, values, 1)
+                          ->setSourceRange(tree->range());
+          Value* output = graph->insertNode(node)->output();
+          output->setType(StringType::get());
+          return output;
+        }
+      }
```

Comment:
I sense that the fallthrough behavior here is intentional, but still better to make it obvious by adding a comment here.

In c++ 17, there is a fallthrough attribute where you can make it really obvious, but Pytorch hasn't adopted C++ 17 yet.